### DISCUSSION TITLE
Suggestion: Move Bank Tabs to Bank  Tags

### DISCUSSION THREAD

**Discussion by LaktosG** [runelite/runelite#17393] [Ideas]
Hi, I just tried doing this because it felt intuitive. Didnt work tho. But it would be cool if you/someone could add a right-click op tion that lets u add and entire -bank tab-(referring to the already existing orsrs bank tabs outside of runelite) into a -bank tag-(referring to the '3rd party' bank tags that is a function/addon in vanilla RuneLite)

this would make new tags easier for example if you wanna put all your gear, potions and food in a single bank tag, like a slayer tag. without the functionality of adding entire tabs to tags you would have to make maybe 50-100+ clicks per tab but it could be reduced to <10 :)

**Comment by raiyni** [comment]
You can tag your inventory and gear by right clicking the deposit icons.
